### PR TITLE
[ome_alert_policies_category_info] : This module retrieves the information on categories of alert policies for OpenManage Enterprise.

### PR DESCRIPTION
**Pull request by jagadeeshnv**

### CODE REVIEW

**Review by sachin-apa — DISMISSED** [review]

File: plugins/modules/ome_alert_policies_category_info.py [hunk 1]
```diff
@@ -0,0 +1,756 @@
...
+
+def get_formatted_categories(rest_obj):
+    resp = rest_obj.invoke_request("GET", ALERT_CATEGORY_URI)
+    categories = remove_key(resp.json_data.get("value", []))
+    return categories
+
+
+def main():
+    specs = {}
+    specs.update(ome_auth_params)
```
Comment (Kritika-Bhateja-03) [inline plugins/modules/ome_alert_policies_category_info.py]: Directly assign specs = ome_auth_params

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: playbooks/ome/alert_policy/ome_alert_policies_category_info.yml
Comment (ABHISHEK-SINHA10) [inline playbooks/ome/alert_policy/ome_alert_policies_category_info.yml:1]: remove connection: local and add delegate_to: localhost to each task

**Comment by sachin-apa** [comment]
@jagadeeshnv Readme in plugin directory need an update to add the module entry, rest looks good ..

**Review by ShivamSh3 — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by Kritika-Bhateja-03 — APPROVED** [review]

**Pull request merged**